source/scss/_pf-component-card.scss

```diff
@@ -17,13 +13,94 @@
   .card-header{
     background: $pf-card-cap-bg;
     border-bottom-color: $pf-card-border-color;
+    padding: $pf-spacer-lg;
+    .card-title {
+      margin: 0;
+    }
+    .dropdown {
+      margin: -$pf-spacer-xxs 0;
+    }
   }
-
   .card-footer{
     background: $pf-card-footer-bg;
-    border-top-color: $pf-card-border-color
+    border-top-color: $pf-card-border-color;
+    padding: $pf-spacer-sm $pf-spacer-lg;
+    p {
+      margin: $pf-spacer-xxxs 0;
+    }
+  }
+  .card-block {
+    padding: $pf-spacer-lg;
   }
+  .card-title {
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
Lets talk about it. My idea is to add something like an `.pf-card__title--ellipsis` that cuts the title. Not everyone will want to cut the text.
But lets talk about the best way to do it fi you want 😄